src/include/utime.h

```diff
@@ -433,23 +438,23 @@ class utime_t {
 WRITE_CLASS_ENCODER(utime_t)
 WRITE_CLASS_DENC(utime_t)
 
-
 // arithmetic operators
 inline utime_t operator+(const utime_t& l, const utime_t& r) {
-  return utime_t( l.sec() + r.sec() + (l.nsec()+r.nsec())/1000000000L,
-                  (l.nsec()+r.nsec())%1000000000L );
+  __u64 sec = (__u64)l.sec() + r.sec() + (l.nsec()+r.nsec())/1000000000ul;
+  __u64 nsec = (l.nsec()+r.nsec())%1000000000ul;
+  return utime_t( ut_safe_convert(sec), ut_safe_convert(nsec));
```

Comment:
is the second ut_safe_convert() necessary?

remove the space after `(`.

why you are using different ways to normalize the result in `operator+` and `operator+=`? could you be more consistent ?